Library/Homebrew/test/system_command_spec.rb

```diff
@@ -196,11 +196,14 @@
       ] }
     }
 
+    # rspec-wait wraps the expectation
+    # rubocop:disable RSpec/NoExpectationExample
     it "returns without deadlocking" do
       wait(30).for {
```

Comment:
Side note: from a cursory glance, this may be the sole use of the `rspec-wait` gem, which is required in every `rspec` run via `spec_helper`.

Reply:
@dduugg Good catch! If you can figure out an implementation that allows removing that gem: it'd be a great PR!